flink-runtime/src/main/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGenerator.java

```diff
@@ -1259,7 +1259,7 @@ public static void setOperatorConfig(
         config.setCheckpointMode(StreamGraph.getCheckpointingMode(checkpointCfg));
         config.setUnalignedCheckpointsEnabled(checkpointCfg.isUnalignedCheckpointsEnabled());
         config.setUnalignedCheckpointsSplittableTimersEnabled(
-                checkpointCfg.isUnalignedCheckpointsInterruptibleTimersEnabled());
+                checkpointCfg.isUnalignedCheckpointsSplittableTimersEnabled());
```

Comment:
yes - it is a typo and should be changed - can you add a test to ensure setOperatorConfig is setting the config as expected please.